frontend/src/components/config-editor.ts

```diff
@@ -96,7 +92,7 @@ export class ConfigEditor extends LiteElement {
             autocomplete="off"
             autocapitalize="off"
             spellcheck="false"
-            wrap="off"
+            wrap="${"off" as any}"
```

Comment:
Curious why this is necessary?

Reply:
The value "off" is non-standard and not supported by any browsers except Firefox, so lit-analyzer (rightfully) sees it as an error